.setup-CF.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+echo Entering "$(cd "$(dirname "$0")" && pwd -P)/$(basename "$0")" in `pwd`
+
+# Fail the whole script if any command fails
+set -e
+
+export SHELLOPTS
+
+export JAVA_HOME=${JAVA_HOME:-$(dirname $(dirname $(readlink -f $(which javac))))}
+
+git -C /tmp clone --depth 1 -q https://github.com/plume-lib/plume-scripts.git
+
+export JSR308="${JSR308:-$(cd .. && pwd -P)}"
```

Comment:
```suggestion
```

I think this variable isn't used anywhere. Let me commit this and see whether tests pass. If they do, please do the same in the other repos.